kDrive/UI/Controller/Menu/SwitchUserViewController.swift

```diff
@@ -158,12 +169,11 @@ extension SwitchUserViewController: InfomaniakLoginDelegate {
     func didCompleteLoginWith(code: String, verifier: String) {
         Task {
```

Comment:
This probably runs as well in the mainthread given the UIViewController inheritance.